### PR TITLE
[LimitOrder] Limit order mock

### PR DESCRIPTION
**Pull request by anxolin**
Part of #1381

This PR creates a mock up for a new component `LimitOrder` that will be the factored version of `TradeWidget`.

The mockup defines clearly the subcomponents that will have, and provides a very basic form of them. Next PRs can start refactoring the existing ones, so the real ones can replace the fake ones.

Inlcudes
* New component `LimitOrder`, with the basic implementation (kind of mocked) parts fo TradeWidget
* Stories to show some parametrization, and some small logic like invert price
* Simplifies SwapPrice component. It had some unused props

image

### CODE REVIEW

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr1382--dexreact.review.gnosisdev.com

**Review by W3stside — APPROVED** [review]
Nice, works well for me

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from '@storybook/react/types-6-0'
+import { LimitOrder, Props } from './LimitOrder'
+import BigNumber from 'bignumber.js'
+
+export default {
+  title: 'Trade/LimitOrder',
```
Comment (Velenir) [inline src/components/trade/LimitOrder/LimitOrder.stories.tsx:9]: :book: :cry:

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    />
+  )
+}
+
+export const Basic = Template.bind({})
+Basic.args = {}
+
+export const NoAmount = Template.bind({})
+NoAmount.args = {
+  amount: undefined,
```
Comment (Velenir) [inline src/components/trade/LimitOrder/LimitOrder.stories.tsx:56]: why `undefined`, just not passing not an option?
or does this remove `amount control` or something?

**Review by Velenir — APPROVED** [review]

**Review by Velenir — CHANGES_REQUESTED** [review]
Broken: 
https://pr1382--dexreact.review.gnosisdev.com/storybook/?path=/story/trade-price--basic
https://pr1382--dexreact.review.gnosisdev.com/storybook/?path=/story/common-swapprice--no-args

**Comment by anxolin** [comment]
> Broken:
https://pr1382--dexreact.review.gnosisdev.com/storybook/?path=/story/trade-price--basic
https://pr1382--dexreact.review.gnosisdev.com/storybook/?path=/story/common-swapprice--no-args

Yep, the swap token I've made a separate PR: https://github.com/gnosis/dex-react/pull/1388
For the price one, the plan is do it in another isolated PR too:
image

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx (hunk 2 above)
Comment (anxolin) [reply src/components/trade/LimitOrder/LimitOrder.stories.tsx:56]: I defined some defaults for a "normal" case. Then I just pass undefined to override that

File: src/components/trade/LimitOrder/LimitOrder.stories.tsx (hunk 1 above)
Comment (anxolin) [reply src/components/trade/LimitOrder/LimitOrder.stories.tsx:9]: I will add it in any missing story in this PR

**Review by Velenir — APPROVED** [review]

**Pull request merged**